home.hbs

```diff
@@ -27,112 +27,107 @@
+                <img  class="hero-logo img-fluid" src="{{asset "images/logo.png"}}">
+            </div>
+            <div class="text-light text-center font-weight-bold h1 hero-text mr-5 ml-5 pr-5 pl-5"> {{#page}} {{content}} {{/page}} </div>
+            <div data-aos="fade-up" class="d-flex flex-row justify-content-center">
+                <a class ="pr-4 pl-4" href="https://www.facebook.com/wpgpoliceharm/" target="_blank"> 
+                    <img class="hvr-float"  src="{{asset "images/facebook.png"}}" width="50" height="50"> </a>
+                <a class ="pr-4 pl-4" href="https://www.instagram.com/wpgpoliceharm/" target="_blank"> 
+                    <img class="hvr-float"  src="{{asset "images/instagram.png"}}" width="50" height="50"> </a>
+                <a class ="pr-4 pl-4" href="https://twitter.com/WpgPoliceHarm" target="_blank"> 
+                    <img class="hvr-float"  src="{{asset "images/twitter.png"}}" width="50" height="50"> </a>
             </div>
             
             <!--<section><h3>Recent posts from the blog</h3><ul><li> <a href="/blog/the-most-recent-post-so-far">The most recent post so far</a></li><li> <a href="/blog/a-simple-post">A simple blog post</a></li><li> <a href="/blog/another-post">Another post</a></li></ul></section>-->
         </div>
     </div>
     <div id="mission" class="section-1">
-
-        <div data-aos="fade-up" class="text-heading mission-text-heading" > <h2>OUR MISSION</h2></div>
-        <div data-aos="fade-up"> <div class="text-body"> {{#page}} {{content}} {{/page}} </div></div>
-
-        <br>
-        <div data-aos="fade-up" class="text-heading mission-text-heading" > <h2>OUR VISION</h2></div>
-        <div data-aos="fade-up" > <div class="text-body"> We are a leaderless, horizontally organized group that is anti-capitalist, anti-carceral, anti-colonial, and anti-oppressive. Our approach to our work is intersectional. We are in humble solidarity with all peoples and communities who are harmed by the WPS. <br> <br> We do not advocate for police reform, but rather the defunding and abolition of the WPS and reallocation of their funds to sustainable, community-led alternatives. We strive to apply these values in our organizing work, our interactions with each other, and in our personal lives. </div></div>
+        <div data-aos="fade-up" class="text-heading mission-text-heading " > 
+            <h2>OUR VISION</h2>
+        </div>
+        <div data-aos="fade-up" > 
+            <div class="text-dark font-weight-bold h1 mx-auto hero-text text-center" style="width:75%"> We are a leaderless, horizontally organized group that is <span class="text-danger">anti-capitalist</span>,  <span class="text-danger">anti-carceral</span>, <span class="text-danger">anti-colonial</span>, and <span class="text-danger">anti-oppressive</span>. Our approach to our work is intersectional. We are in humble solidarity with all peoples and communities who are harmed by the Winnipeg Police Service (WPS).

```

Comment:
I feel like it can be confusing to use a different colour for emphasis because that’s often what happens for links. What do you think about returning to the default font weight (and non-italics) for this so bold can be used for emphasis instead?

![image](https://user-images.githubusercontent.com/43280/103236925-d0b62d00-490b-11eb-9384-bd50ab27dca9.png)

Also for this underline:

![image](https://user-images.githubusercontent.com/43280/103237012-06f3ac80-490c-11eb-9304-0915f9863fd8.png)

Reply:
Oh yeah, I didn't consider that. Great idea! Going to make these changes today.